cocos/2d/components/sprite.ts

```diff
@@ -686,6 +688,22 @@ export class Sprite extends UIRenderer {
             }
         }
     }
+
+    /**
+     * @engineInternal
+     */
+    public _updateBlendFunc () {
```

Comment:
Where does it been invoked?

Reply:
`Sprite` inherited from `UIRenderer`,  `UIRenderer._updateBlendFunc` will be invoked in `UIRenderer.updateMaterial`.